requirements_tools/upgrade_requirements.py

```diff
@@ -78,8 +79,13 @@ def installed(requirements_file):
         return expected_pinned
 
 
-def dirs(tmp):
-    dirnames = ('venv', 'venv/bin/python', 'venv/bin/pip')
+def venv_paths(tmp, pip_tool):
+    dirnames = (
+        'venv',
+        'venv/bin/python',
+        'venv/bin/pip',
+        'venv/bin/' + pip_tool
```

Comment:
maybe this function should `shlex.split` `pip_tool` (since all callers seem to do that anyway)

Reply:
I thought about it, but it complicates the rest of the function.  Not so easy to join paths when one of the paths is a list.